tests/virt/cluster/vm_cloning/utils.py

```
@@ -33,20 +37,30 @@ def check_if_files_present_after_cloning(vm):
 
-def assert_target_vm_has_new_pvc_disks(source_vm, target_vm):
-    def _get_data_volumes_list(vm):
+def assert_target_vm_has_new_pvc_disks(source_vm: VirtualMachine, target_vm: VirtualMachine, prefix: str) -> None:
+    def _get_data_volumes_list(vm: VirtualMachine) -> list[str]:
         return [volume["dataVolume"]["name"] for volume in vm.vmi.instance.spec.volumes if "dataVolume" in dict(volume)]
 
-    LOGGER.info("Checking that the target VM created new DataVolumes")
+    def _get_disk_names_list(vm: VirtualMachine) -> list[str]:
+        return [volume["name"] for volume in vm.vmi.instance.spec.volumes if "dataVolume" in dict(volume)]
+
+    LOGGER.info(f"Checking that the target VM created new DataVolumes with unique names and names with prefix {prefix}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)